app/renderers/holding_location_renderer.rb

```diff
@@ -0,0 +1,14 @@
+class HoldingLocationRenderer < CurationConcerns::AttributeRenderer
+  def initialize(value, options = {})
+    super(:location, value, options)
+  end
+
+  private
+
+    def attribute_value_to_html(value)
```

Comment:
Wouldn't it be more efficient to change the API to `location_service = HoldingLocationService.new(value)` (or .find), and then have instance methods for email/label/phone?